src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -4691,12 +4696,20 @@ def test_aks_browse_legacy(self, resource_group, resource_group_location):
         except subprocess.CalledProcessError as err:
             raise CLIInternalError("Failed to install kubectl with error: '{}'!".format(err))
 
+        # create test hook file
+        hook_file_path = self._get_test_data_file("test_aks_browse_legacy_hook.json")
+        with open(hook_file_path, "w") as f:
+            json.dump("enabled", f)
+
         # test aks browse cmd
         browse_cmd = 'aks browse --resource-group={resource_group} --name={name} --listen-address=1.1.1.1 --listen-port=8080 --disable-browser'
         self.cmd(browse_cmd, checks=[
             StringCheck("Test Invalid Address! Test Passed!")
         ])
 
+        # delete test hook file
```

Comment:
try catch finally.

Reply:
changed in commit [0acf691](https://github.com/Azure/azure-cli/pull/19115/commits/0acf691b14b97f346e078c96d44cc2c4f7215c45).